homeassistant/components/unifi/controller.py

```diff
@@ -63,6 +65,16 @@ def block_clients(self):
         """Return list of clients to block."""
         return self.unifi_config.get(CONF_BLOCK_CLIENT, [])
 
+    @property
+    def dont_track_clients(self):
+        """Return if clients should be tracked."""
+        return self.unifi_config.get(CONF_DONT_TRACK_CLIENTS, False)
```

Comment:
Don't add the default here, add it to the config schema and then just get it by indexor.